data/categories.txt

```diff
@@ -638,6 +638,46 @@ sk:3Banka
 sw:Benki|fedha
 fa:بانک
 
+# First keyword should match [secondhand] definition in strings.txt!
+@secondhand
+en:6Second Hand
+ar:اليد الثانية
+be:Другая рука
+bg:Втора употреба
+ca:Segona mà
+cs:Z druhé ruky
+da:Brugt
+de:Second-Hand
+el:Δεύτερο χέρι
+es:Segunda mano
+et:Second hand
+eu:Bigarren eskuko
+fa:دست دوم
+fi:Second hand
+fr:Deuxième main
+he:יד שניה
+hu:Second hand
+id:Tangan kedua
+it:Di seconda mano
+ja:セカンドハンド
+ko:초침
+mr:वापरलेले
+nb:Brukt
+nl:Tweedehands
+pl:Z drugiej ręki
+pt:Segunda mão
+ro:La mâna a doua
+ru:Вторая рука
+sk:Z druhej ruky
+sv:Andra handen
+sw:Mtumba
+th:มือสอง
+tr:İkinci el
+uk:Секонд-хенд
+vi:Đồ cũ
+zh-Hans:二手货
+zh-Hant:二手
```

Comment:
be, ru have strange translation (literally like man's second hand) and I suspect the same on others languages

Reply:
fixed